oauth/scopes.py

```diff
@@ -0,0 +1,90 @@
+CREATE = "create"
```

Comment:
I'd like to see comments here describing what each section means/does

Reply:
I don't believe this is necessary with the addition of meaningful `SCOPE_DOCS`, but let me know if you disagree.